### ISSUE TITLE
Investigate and fix mobile nav menu icon button getting small and log in menu button wrapping

### ISSUE THREAD

**Issue by bendansby** [NoRedInk/noredink-ui#1218]
![717871BF-0615-46E1-8F7A-D9F7024D8CCE_1_105_c](https://user-images.githubusercontent.com/13528834/210423283-21dc03bf-38b5-4cc6-9f20-843b9304f4e9.jpeg)

**Comment by tesk9** [comment]
The Log-in wrapping is pre-existing, so I'm not going to fix that, but I'm addressing the clickable svg resizing behavior

**Comment by bendansby** [comment]
Oh I guess there's more space between the hamburger and logo now for some reason, causing Log In to wrap? Or something?

**Comment by tesk9** [comment]
I think it's been wrapping? at least at 320px, I think it's always wrapped. I referenced demo, since prod had already been deployed, and it is definitely wrapping there.

As far as I understand things, the navbar was relying on the hamburger and logo to shrink, and I think I made it even harder to achieve flex-shrinking on the noredink-ui side. (Because usually I mess things up the over-shrinking direction!)
